modules/time.js

```diff
@@ -0,0 +1,10 @@
+export default class TimeDate {
+  constructor() {
+    this.time = document.querySelector('.time');
+  }
+
+  setDate() {
+    const date = new Date();
+    this.time.innerHTML = date.toLocaleString('en-US');
+  }
+}
```

Comment:
- [ ] Kindly make sure you use the [Luxon](https://github.com/moment/luxon/blob/master/docs/install.md) library to get the **DateTime**. You can see :point_right: :point_right:  [Get the current date and time](https://github.com/moment/luxon/blob/master/docs/tour.md#get-the-current-date-and-time) here to have more information.
   * I can suggest you add the package as a `.js` file to the project (in full or minified version)
   * And then import Module `import { DateTime } from "luxon";` and use the the `DateTime` where it is expected to be used.